src/navigator/DrawerNavigator.tsx

```diff
@@ -216,6 +216,9 @@ export default function DrawerNavigator({ route }: Props) {
   const { showAddWithdrawOnMenu, showSwapOnMenu } = getExperimentParams(
     ExperimentConfigs[StatsigExperiments.HOME_SCREEN_ACTIONS]
   )
+  const { dappVerbiageEnabled } = getExperimentParams(
+    ExperimentConfigs[StatsigExperiments.DAPP_VERBIAGE]
+  )
```

Comment:
nit: Could there be more meaningful names?
so it's easier to understand what it does.
```suggestion
  const { discoverCopyEnabled } = getExperimentParams(
    ExperimentConfigs[StatsigExperiments.DAPPS_MENU_ITEM_COPY]
  )
```

Reply:
@Jean adjusted the experiment name and params in https://github.com/valora-inc/wallet/pull/3875/commits/dc54e9dc7548ac6cf2ae8e6924d99d58600c992c!